share/ci/install/isaac.sh

```diff
@@ -19,7 +19,7 @@ if [ -z "$DISABLE_ISAAC" ] ; then
     cd $CI_PROJECT_DIR
     git clone https://github.com/ComputationalRadiationPhysics/isaac.git
     cd isaac
-    git checkout 195420ca1c43f6148c7f6c3a10ad9cad32e04d6a
+    git checkout 6bf8e189142da7ab16f8367316fb21dcaa7e3b78
```

Comment:
@psychocoderHPC why do you specify a specific branch (the current head?)

Reply:
*offline discussion with @psychocoderHPC *: This avoids that the PIConGPU CI breaks because we accidentally introduce an error in ISAAC.